sdk/core/src/CommonNetworkMember/BaseNetworkMember/AnchoringService.ts

```diff
@@ -0,0 +1,80 @@
+import { ParsedOptions } from '../../shared/getOptionsFromEnvironment'
+import { extractSDKVersion } from '../../_helpers'
+import { ELEM_DID_METHOD, JOLO_DID_METHOD } from '../../_defaultConfig'
+import { RegistryApiService } from '@affinidi/internal-api-clients'
+import { KeysService } from '@affinidi/common'
+
+export class AnchoringService {
+  constructor(
+    private readonly encryptedSeed: string,
+    private readonly password: string,
+    private readonly didDocument: any,
+    private readonly nonce: number,
+    private readonly options: ParsedOptions,
+    private readonly anchoredDidElem?: boolean,
```

Comment:
But `anchoredDidElem` doesn't seem to be used anywhere?

Reply:
used in anchorDidWithElemMethod